### PR TITLE
New package: 2048-qt

### PR DESCRIPTION
**Pull request by cr6git**
Based on [PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=2048-qt).

### CODE REVIEW

File: srcpkgs/2048-qt/template [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+homepage="https://github.com/xiaoyong/2048-Qt"
+license="MIT"
+distfiles="https://github.com/xiaoyong/2048-Qt/archive/v${version}.tar.gz"
+checksum=b895ecdbc09127215baf2633b513726cb39e24f22cc309ec390ac9bdae9b224c
+wrksrc="2048-Qt-${version}"
+
+do_install() {
+	vbin 2048-qt
+
+	sizes=(16 32 48 256)
```
Comment (chneukirchen) [inline srcpkgs/2048-qt/template]: Don't use bash arrays, just inline this in the `for` loop below.

File: srcpkgs/2048-qt/template [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	vinstall icons/scalable/apps/2048-qt.svg 644 usr/share/icons/hicolor/scalable/apps
+
+	vlicense LICENSE
+
+	vinstall 2048-qt.desktop 644 usr/share/applications
+	vdoc README.md
+	vdoc Changelog.md
+
+	vman man/2048-qt.6
+	gzip ${DESTDIR}/usr/share/man/man6/2048-qt.6
```
Comment (chneukirchen) [inline srcpkgs/2048-qt/template]: Don't compress man pages.

File: srcpkgs/2048-qt/template [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Template file for '2048-qt'
+pkgname=2048-qt
+version=0.1.5
+revision=1
+build_style=qmake
+hostmakedepends="qt5-declarative-devel"
+makedepends="qt5-declarative-devel"
+depends="desktop-file-utils hicolor-icon-theme qt5-quickcontrols xdg-utils"
```
Comment (chneukirchen) [inline srcpkgs/2048-qt/template]: qt5-quickcontrols should be inferred, no?

Comment (cr6git) [inline srcpkgs/2048-qt/template]: For the translation: inferred = removed , right ? 
if I remove `qt5-quickcontrols` from `depends=` I can't run the program, I get this error: 
````
QQmlApplicationEngine failed to load component
qrc:///qml/main.qml:4 module "QtQuick.Dialogs" is not installed
qrc:///qml/main.qml:2 module "QtQuick.Controls" is not installed
qrc:///qml/main.qml:3 module "QtQuick.Controls.Styles" is not installed
qrc:///qml/main.qml:4 module "QtQuick.Dialogs" is not installed
qrc:///qml/main.qml:2 module "QtQuick.Controls" is not installed
qrc:///qml/main.qml:3 module "QtQuick.Controls.Styles" is not installed
qrc:///qml/main.qml:4 module "QtQuick.Dialogs" is not installed
qrc:///qml/main.qml:2 module "QtQuick.Controls" is not installed
qrc:///qml/main.qml:3 module "QtQuick.Controls.Styles" is not installed
````

Comment (pullmoll) [inline srcpkgs/2048-qt/template]: qt5-quickcontrols is no library, but just qml scripts and images.
If a package needs it, it has to be listed in depends.

Comment (cr6git) [inline srcpkgs/2048-qt/template]: I wonder why my PR is stalled.

**Comment by cr6git** [comment]
Nice!

**Pull request merged**